content/docs/gke/gcp-e2e.md

```diff
@@ -89,768 +89,29 @@ In summary:
 
 It's time to get started!
 
-## Set up your environment
+### Run the MNIST tutorial on GCP
 
-### Download the project files
+1. Follow the [GCP instructions](https://www.kubeflow.org/docs/gke/deploy/) to deploy Kubeflow with IAP
```

Comment:
Add a full stop (period) to the end of this sentence and others lower down in the doc.